pages/index.js

```diff
@@ -1,55 +1,36 @@
 import Head from 'next/head';
-import { useState } from 'react';
-import Articles from './components/Articles';
-import Sources from './components/Sources';
+import Articles from '../components/Articles';
+import Sources from '../components/Sources';
```

Comment:
Do not put `/components` inside `pages` folder. It makes the components accessible on `baseUrl/components/ComponentName` path in the browser. tdlr; it will be treated as a page on components path.

Reply:
So I moved the `components` folder to the top level.